cmd/minikube/cmd/start.go

```diff
@@ -1030,8 +1030,18 @@ func validateNetwork(h *host.Host, r command.Runner) string {
 
 func trySSH(h *host.Host, ip string) {
 	sshAddr := fmt.Sprintf("%s:22", ip)
-	conn, err := net.Dial("tcp", sshAddr)
-	if err != nil {
+
+	dial := func() (err error) {
+		dialer := net.Dialer{Timeout: 3 * time.Second}
```

Comment:
lets rename this to d, since it is such a short lived variable

Reply:
Done, thanks for tip